dotcom-rendering/src/web/components/DynamicSlowMPU.tsx

```diff
@@ -277,3 +275,65 @@ export const DynamicSlowMPU = ({
 		}
 	}
 };
+
+const RenderNonMPU = ({ trails, containerPalette, showAge }: NonMPUProps) => {
```

Comment:
this can be seen in Georges' screenshots in the PR description, but just to clarify: you might expect DynamicSlowMPU to use the DynamicSlow layout if the request is ad free, but that's not what Frontend does so we aren't doing it here either. Perhaps something to look into in the future, though?